src/main/scala/no/ndla/articleapi/model/api/TSTypes.scala

```diff
@@ -0,0 +1,15 @@
+package no.ndla.articleapi.model.api
+
+import com.scalatsi._
+import no.ndla.articleapi.model.domain
+
+/**
+  * The `scala-tsi` plugin is not always able to derive the types that are used in `Seq` or other generic types.
+  * Therefore we need to explicitly load the case classes here.
+  * This is only necessary if the `sbt generateTypescript` script fails.
+  */
+object TSTypes {
+  implicit val relatedContent: TSIType[RelatedContentLink] = TSType.fromCaseClass[RelatedContentLink]
+  implicit val availability: TSType[domain.Availability.Value] =
+    TSType.sameAs[domain.Availability.Value, domain.Availability.type]
```

Comment:
Stemmer for at vi hiver inn lenker til issuene i en kommentar sånn at det blir enklere å sjekke om de har blitt fikset i nyere versjoner når vi oppgraderer scala-tsi i fremtiden:

```suggestion
  // Type-aliases referencing generics doesn't not work without this in scala-tsi. See: https://github.com/scala-tsi/scala-tsi/issues/184
  implicit val relatedContent: TSIType[RelatedContentLink] = TSType.fromCaseClass[RelatedContentLink]
  // Scala2 enumerations doesn't work as expected in scala-tsi. See: https://github.com/scala-tsi/scala-tsi/issues/182
  implicit val availability: TSType[domain.Availability.Value] =
    TSType.sameAs[domain.Availability.Value, domain.Availability.type]
```